docs/pages/develop/unit-testing.mdx

```diff
@@ -40,6 +40,8 @@ A starting configuration you can use is to make sure any modules you are using w
   "preset": "jest-expo",
   "transformIgnorePatterns": [
     "node_modules/(?!((jest-)?react-native|@react-native(-community)?)|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|react-native-svg)"
+    // In case you are using PNPM as package manager, use this instead: 
+    // `node_modules/(?!(?:.pnpm/)?((jest-)?react-native|@react-native(-community)?|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|react-native-svg))`
```

Comment:
```suggestion
    // "node_modules/(?!(?:.pnpm/)?((jest-)?react-native|@react-native(-community)?|expo(nent)?|@expo(nent)?/.*|@expo-google-fonts/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|react-native-svg))"
```

I think it is safe to use double quotes instead of back ticks since this is a JSON file and if someone decides to copy from the snippet and just un comment the line.